### PR TITLE
Pull request for alkis ontology

### PR DESCRIPTION
**Pull request by d3cebal**

### CODE REVIEW

File: alkis/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# ALKIS
```
Comment (davidlehn) [inline alkis/README.md]: Please add at least maintainer info (like github user name) here, thanks.

File: alkis/.htaccess [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+RewriteEngine on
+
+#Rewrite rules for ALKIS vocabulary extension
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^alkis$ https://github.com/tubav/alkis/blob/master/  [R=303,NE,L]
```
Comment (davidlehn) [inline alkis/.htaccess]: Rules are relative to current directory, so `^alkis$` would make the end point https://w3id.org/alkis/alkis.  You probably just want `^$`.  Same below.

File: alkis/README.md (hunk 1 above)
Comment (d3cebal) [reply alkis/README.md]: Done.

File: alkis/.htaccess (hunk 2 above)
Comment (d3cebal) [reply alkis/.htaccess]: Also done :)

**Pull request merged**